docs/data/date-pickers/fields/fields.md

```diff
@@ -37,25 +37,29 @@ The single input range fields are not supported on the range pickers yet (you ca
 
 The field components have an internal state to update the visible value.
 
-It will only call the `onChange` callback when all the sections of the modified date are filled.
+It will only call the `onChange` callback when:
+
+- all the sections of the modified date are filled
+- all the sections of the modified date are empty (will return `null` for the modified date)
+- all the sections of the modified date were filled, and the user cleaned the content of one section (will return `Invalid date` for modified date)
 
 #### On single element fields
 
-On a single date field (`DateField` / `DateTimeField` / `TimeField`),
-`onChange` will be called if all the sections are filled.
+On a single item field (`DateField` / `DateTimeField` / `TimeField`),
```

Comment:
'item' is confusing. I first though you were talking about a field with only year for example

Reply:
I'm trying to avoid the term "date" or "value" since they describe something else.
Do you have another idea ?

Reply:
For MIT components?

Reply:
I don't understand what you mean.

Reply:
I was thinking MIT vs Pro, but not a great idea.

Maybe "On a simple field" which avoids the notion of number associated with "single"

Reply:
Could work indeed :+1: 
I changed on my PR